VERSION_NUMBER

```diff
@@ -1 +1 @@
-1.12.0
```

Comment:
Oops :-)
I'll send a fix.

Reply:
Thank you! This is fixed in `main` branch but I didn't touch release branch.